wallet/address.go

```diff
@@ -0,0 +1,67 @@
+package wallet
+
+import (
+	"encoding/hex"
+	"fmt"
+	"perun.network/go-perun/wallet"
+)
+
+// PubKeyLength is the length of the public verification key part of a Cardano `ed25519` keypair in bytes.
+const PubKeyLength = 32
+
+// TODO: the Address implementation of wallet.Address only represents the public verification key.
```

Comment:
TODO comments should not be merged. Transform into issue or remove if resolved ;)

Reply:
It was transformed into an issue (#2) - nice to see that you're still keeping an eye on us :D